src/sets.jl

```diff
@@ -1,16 +1,14 @@
 export incl, swap
 
-import Base: Set
+const pm_Set_suppT = Union{Int32, Int64}
```

Comment:
Wasn't there set of sets at some point?

Reply:
no, there are `pm_Array`s of `pm_Set`s of `int`s; `pm_Set`s are only of `int` and `(long) long` ;)

Reply:
ah okay :)